lib/internal/process/next_tick.js

```diff
@@ -55,8 +54,8 @@ function processTicksAndRejections() {
   let tock;
   do {
     while (tock = queue.shift()) {
-      const asyncId = tock[async_id_symbol];
-      emitBefore(asyncId, tock[trigger_async_id_symbol]);
+      const asyncId = tock.asyncId;
```

Comment:
Doesn't this change non-normalize how we do async id properties across other internals?

Reply:
I don't see why it matters. As far as I know the primary reason for using the symbols in all other instances is because that's the easiest way to associate the ids with the resource while making the ids less public. Since the entries in the next tick queue are never exposed to userland, there isn't really a need to use the symbols for hiding the ids.

Reply:
Not strictly true. They’re exposed when the async hook init event fires.